### ISSUE TITLE
Enemies can potentially be placed on top of party members

### ISSUE THREAD

**Issue by pyroll** [gameboy9/FF6KefkaRush#3]
![Pasted image 20220911012034 1](https://user-images.githubusercontent.com/61872672/189543960-a25d4ed1-a56e-4065-963d-fd1573bb7fd1.png)

**Comment by gameboy9** [comment]
Yeah, this is a problem with generating randomized "surrounded" formations... I'm going to try to find a better way to deal with that.
